### ISSUE TITLE
Enums with a single option is never

### ISSUE THREAD

**Issue by suneab** [sinclairzx81/typebox#585]
Starting in version 0.31.9, it is no longer possible to use enums with a single option

```
import { Static, Type } from "@sinclair/typebox"
enum CommunicationType {
    email = "email
}
const typeboxUser = Type.Object({
    name: Type.String(),
    type: Type.Enum(CommunicationType),
})
type User = Static<typeof typeboxUser>
const user:User = {
    name: "Suneab",
    type: CommunicationType.email
}
console.log(user)
```

Interface resolves:
```
type User = {
    name: string;
    type: never;
}
```
Adding a second option, like `sms = "sms"` will make the schma act as expected
```
type User = {
    name: string;
    type: CommunicationType;
}
```

A workaround is to use 
`Type.Union([Type.Literal(CommunicationType.email)]),`

**Comment by sinclairzx81** [comment]
@suneab Hi

This was actually resolved on 0.31.13, however this new Enum implementation has been reverted as of last night due to scalability issues. You can read a full history of updates relating to enum bugs and challenges on this issue https://github.com/sinclairzx81/typebox/issues/574

If you install 0.31.14, things should be back to the working 0.31.8 implementation of Enum.
Let me know how you go.
Cheers
S

**Comment by suneab** [comment]
@sinclairzx81 Hi
I was looking at this yesterday and I didnt get around to post the issue until today... I didnt realise you fixed it already 🤦 
Thanks a lot!
